packages/harpguru-core/src/global.d.ts

```diff
@@ -13,5 +13,40 @@ declare module 'reactn/default' {
     fragmentHarpFaceByOctaves: boolean
     flushChannel: FlushChannels
     activeQuizDegrees: ReadonlyArray<DegreeIds>
+    // columnBounds: 'FIT' | [ number, number ]
   }
 }
+
+// Introduction of columnBounds
```

Comment:
Raised as techdebt in #113 

The work done to make the app sizing dynamic (at #107) was used as a platform for additional speculative pull requests to ensure that this was the correct direction to start down.

Rather than addressing the finer details in this or subsequent pull request and potentially struggling to merge up to those subsequent branches I will prefer to record the issues in each merge request and resolve them afterwards.

This has a limit of course. Major issues will not be merged to master. I will need to find a way around that if it happens.